src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -107,11 +119,17 @@ namespace ngraph
 
                     // Map attribute values to nGraph enums
                     const auto compute_max = static_cast<bool>(largest);
-                    const auto sort_type = sorted ? ngraph::op::TopK::SortType::SORT_VALUES
-                                                  : ngraph::op::TopK::SortType::NONE;
-
-                    std::shared_ptr<ngraph::Node> top_k = std::make_shared<ngraph::op::TopK>(
-                        data, k, axis, element::i64, compute_max, sort_type);
+                    const auto sort_type = sorted ? ngraph::op::v1::TopK::SortType::SORT_VALUES
+                                                  : ngraph::op::v1::TopK::SortType::NONE;
+
+                    std::shared_ptr<ngraph::Node> top_k = std::make_shared<ngraph::op::v1::TopK>(
+                        data,
+                        k,
+                        axis,
+                        compute_max ? ngraph::op::v1::TopK::Mode::MAX
```

Comment:
Let's move this ternary operator up to keep consistent with `sort_type`

Reply:
Done